devel/gk/Portfile

```
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    gk
+version                 1.0.6
+
+categories              devel
+platforms               darwin
+license                 CC-BY-3.0-US
+installs_libs           no
+maintainers             sergiolms \
+                        alejandroSuch \
+                        JBUinfo
+
+PortGroup               github 1.0
+
+github.setup            gitkraken gk-cli ${version} v
+github.tarball_from     releases
+supported_archs         x86_64 arm64
+
+
+description             Official CLI for GitKraken
+long_description        ${name} is GitKraken on the command line. It makes working across multiple \
+                        repos easier with Workspaces, provides access to pull requests and \
+                        issues from multiple services (GitHub, GitLab, Bitbucket, etc.), \
+                        and seamlessly connects with GitKraken Client and GitLens in VS Code \
+                        to visualize git information when you need it.
+homepage                https://github.com/gitkraken/gk-cli
+
+checksums               ${name}_${version}_macOS_arm64.zip \
+                            sha256  2c9d8b920b781321cf4d40db502f1b8049a2074e41ba9fa8c2755ee31927e052 \
+                            rmd160  42f45dd008fd05407bed976c5b8c82fb70436aab \
+                            size    8872315 \
+                        ${name}_${version}_macOS_x86_64.zip \
+                            sha256  7ac2488301ce2b294aaba9a1bc128da52dbe18eca06803351a078c0b12fa5f0e \
+                            rmd160  9301e54fc86ef3f9f37b80b960fa4793112568e1 \
+                            size    9378365
+
+build                   {}
+use_configure           no
+use_zip                 yes
+
+if { ${os.arch} eq "arm64" } {
```

Please use `${build_arch}` instead of `${os.arch}`:

```suggestion
if { ${build_arch} eq "arm64" } {
```